install/helm/open-match/templates/demo.yaml

```diff
@@ -37,43 +38,52 @@ metadata:
   labels:
     app: {{ template "openmatch.name" . }}
     component: demo
-    chart: {{ template "openmatch.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    {{- include "openmatch.chartmeta" (set . "indent" 4) }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.openmatch.demo.replicas }}
   selector:
     matchLabels:
       app: {{ template "openmatch.name" . }}
       component: demo
-      chart: {{ template "openmatch.chart" . }}
-      release: {{ .Release.Name }}
-      heritage: {{ .Release.Service }}
+      {{- include "openmatch.chartmeta" (set . "indent" 6) }}
   template:
     metadata:
+      namespace: {{ .Release.Namespace }}
+      annotations:
+        {{- include "prometheus.annotations" (dict "port" .Values.openmatch.demo.http.port "prometheus" .Values.openmatch.monitoring.prometheus) | nindent 8 }}
       labels:
         app: {{ template "openmatch.name" . }}
         component: demo
-        chart: {{ template "openmatch.chart" . }}
-        release: {{ .Release.Name }}
-        heritage: {{ .Release.Service }}
+        {{- include "openmatch.chartmeta" (set . "indent" 8) }}
     spec:
+      serviceAccountName: open-match-unprivileged-service
```

Comment:
What is it used for?

Reply:
There's some policies that come along when a service account is bound to the pod. In this case a PodSecurityPolicy that prevents the pod from doing privileged operations.